src/jit/gentree.h

```diff
@@ -925,9 +925,11 @@ struct GenTree
 
 #define GTF_ICON_FIELD_OFF 0x08000000 // GT_CNS_INT -- constant is a field offset
 
-#define GTF_BLK_HASGCPTR 0x80000000  // GT_COPYBLK -- This struct copy will copy GC Pointers
-#define GTF_BLK_VOLATILE 0x40000000  // GT_INITBLK/GT_COPYBLK -- is a volatile block operation
-#define GTF_BLK_UNALIGNED 0x02000000 // GT_INITBLK/GT_COPYBLK -- is an unaligned block operation
+#define GTF_BLK_VOLATILE                                                                                               \
+    0x40000000 // GT_ASG, GT_STORE_BLK, GT_STORE_OBJ, GT_STORE_DYNBLK -- is a volatile block operation
+#define GTF_BLK_UNALIGNED                                                                                              \
+    0x02000000 // GT_ASG, GT_STORE_BLK, GT_STORE_OBJ, GT_STORE_DYNBLK -- is an unaligned block operation
+#define GTF_BLK_INIT 0x01000000 // GT_ASG, GT_STORE_BLK, GT_STORE_OBJ, GT_STORE_DYNBLK -- is an init block operation
```

Comment:
I manually fixed the comment, and then re-ran jit-format. It looks better now.

---
In reply to: [77255368](https://github.com/dotnet/coreclr/pull/7027#discussion_r77255368) [](ancestors = 77255368)